### DISCUSSION TITLE
Number Formatting - Arabic Numbers Format with ar local only - need fix 🤔

### DISCUSSION THREAD

**Discussion by FADL285** [intlify/vue-i18n#1175] [Q&A]
In `Intl.NumberFormat` , when using **ar locale** the numbers are displayed in English format, unlike **ar-EG locale** 

![image](https://user-images.githubusercontent.com/53804570/190882247-87342f5c-0920-412b-bb1d-76d4b6aebf4e.png)

If I want to use **ar** locale in **vue-i18n messages**  and also want to display numbers in Arabic format, is there a way I can achieve this approach without changing the locale to **ar-EG**?
---

### This is Arabic numberFormats for vue-i18n 
![image](https://user-images.githubusercontent.com/53804570/190882775-08b35f31-aa7a-4752-992d-2b118e80dca9.png)

---
### What is currently shown
![image](https://user-images.githubusercontent.com/53804570/190882804-a446a23e-ca2d-4133-8dd3-e7b40a683234.png)

### What I want
![image](https://user-images.githubusercontent.com/53804570/190882828-cbba06c1-27fb-4f15-8b32-ccd2335006f0.png)

**Comment by captainko** [comment]
hi @FADL285 did you find the solution for this?

**Reply by FADL285** [reply]
Not Yet.

**Reply by captainko** [reply]
I have to switch to ar-SA instead

**Reply by FADL285** [reply]
Yes, you should specify the country, e.g., ar-EG, ar-SA. However, it doesn't work if it's just ar.
